src/pages/reporting/ChargebackForm.tsx

```diff
@@ -108,65 +113,81 @@ export default () => {
         initialValues={chargeBackRequest}
         onSubmit={(values) => {
           dispatch(
-            getChargebackData(range, mapPropertiesObjectListToStringOfGuids(values)),
+            getChargebackData(
+              range,
+              mapPropertiesObjectListToStringOfGuids(values),
+            ),
           );
         }}
-        >
+      >
         {(props) => (
-            <Form className="mb-4">
-                <Field
-                    name="groupBy"
-                    component={Select}
-                    options={groupByOptions}
-                    optionLabel="label"
-                    label="Group by"
-                    className="w-100"
-                    placeholder='Choose an option'
-                />
+          <Form className="mb-4">
+            <Field
+              name="groupBy"
+              component={Select}
+              options={groupByOptions}
+              optionLabel="label"
+              label="Group by"
+              className="w-100"
+              placeholder="Choose an option"
+            />
 
-                <h3 className="mt-3">Filter</h3>
+            <b
+              style={{
+                fontSize: '1.25rem',
+                display: 'block',
+                marginTop: '1.5rem',
+              }}
+            >
+              Filters
+            </b>
 
-                {Object.keys(filterByFieldOptions).map((el) => {
-                    const [show, setShow] = useState(false);
-                    return (
-                        <div className='mt-2' key={el}>
-                            <label>{filterByFieldOptions[el].label}</label>
-                            <ToggleButton
-                                className='ml-2'
-                                checked={show}
-                                onChange={({ value }) => {
-                                    setShow(value);
-                                    dispatch(
-                                        value
-                                            ? getPropertyOptions(el, propertyOptions)
-                                            : setPropertyOptions({ ...propertyOptions, [el]: [] }),
-                                    );
+            {Object.keys(filterByFieldOptions).map((el) => {
```

Comment:
By the way, you can change `el` name to `key`
and if you want, move the below code to a separate function